lib/logging.ts

```diff
@@ -57,9 +57,11 @@ function extractErrorInformation (error: Record<'message', string>) {
 }
 
 export function formatLogMessageOneLine<
-  TMessage extends LogMessage | { level: undefined }
+  TMessage extends LogMessage | { level: undefined } | string
```

Comment:
These changes reflect the way contentful-import was using this logging function!